projects/core/src/lib/components/header/header.component.html

```diff
@@ -0,0 +1,52 @@
+<header class="jsmu-header">
+	<nav class="jsmu-header__navigation">
+		<ul class="jsmu-header__avatar">
+			<li class="jsmu-header__avatar-item">
+				<a href="#" class="jsmu-header__avatar-link link">
+					<img
+						src="/assets/images/cat.jpg"
+						alt="profile"
+						class="jsmu-header__avatar-img"
+					/>
+				</a>
+			</li>
+		</ul>
+
+		<div class="jsmu-header__menu-wrapper">
+			<menu class="jsmu-header__menu">
+				<li class="jsmu-header__menu-item list-item" *ngFor="let menuItem of menuItems">
+					<a
+						*ngIf="menuItem.activeOptions; else noOptions"
+						class="jsmu-header__menu-link link"
+						[routerLink]="menuItem.routerLink"
+						[routerLinkActiveOptions]="menuItem.activeOptions"
```

Comment:
add here just [routerLinkActiveOptions]="{ exact: true }" 

and remove from activeOptions from menuItem

Reply:
done